### ISSUE TITLE
Manifest mashup opportunity with Socket Registry

### ISSUE THREAD

**Issue by jdalton** [es-tooling/module-replacements#174]
👋 Socket.dev dev here. Also, the Lodash guy _(please replace it with whatever promotes builtin stuff and simplicity)_. At Socket we've released the [Socket Registry](https://github.com/SocketDev/socket-registry-js) as a place to catalog our growing collection of overrides. We also have a [`@socketsecurity/registry`](https://www.npmjs.com/package/@socketsecurity/registry) package that exposes our [`manifest.json`](https://github.com/SocketDev/socket-registry-js/blob/main/registry/manifest.json). I really like the idea of sharing and where possible consolidating. So figured you might be consume our `manifest.json` or give pointers to what kind of data you'd like tracked to make it more useful for others to consume.

**Comment by 43081j** [comment]
Interesting work you are doing for sure

So if I understood correctly, you do very similar to this repo but we use codemods while you use overrides

The actual detection and manifests are very similar I think

There should be some way we can work together, otherwise it is a lot of duplicated work. I'll have a think but it may also be worth us having a catch up call around this one day

**Comment by jdalton** [comment]
@43081j 
> Interesting work you are doing for sure
>
> So if I understood correctly, you do very similar to this repo but we use codemods while you use overrides

Yes. Codemods are where I'd like to go eventually. While evaluating solutions a few months back the wiring to replace the code wasn't in place _(in this or another project that consumes this)_. The overrides route was a more ready-to-go approach as package managers support it out of the box.

> There should be some way we can work together, otherwise it is a lot of duplicated work. I'll have a think but it may also be worth us having a catch up call around this one day

Great! In the meantime I'm taking inspiration from some of your field names _(like "moduleName")_ and schema stuff _(may help folks consume it)_ to align more closely.

**Comment by gameroman** [comment]
@jdalton @43081j is there anything specific we could do in this repo? I think we can close this for now?

**Comment by jdalton** [comment]
Let's close it for now. Thank you @gameroman!
